### PR TITLE
Adds examples documentation for enabling/disabling tenants.

### PR DESCRIPTION
**Pull request by IsaacCalligeros95**
Adds examples docs for enabling and disabling tenants. At this stage we're not calling it out in the Tenants section, if you think we should let me know.
image

### CODE REVIEW

File: src/shared-content/scripts/enable-disable-tenant-scripts.include.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/tenants"
+)
+
+func main() {
+	apiURL, err := url.Parse("http://localhost:8066")
+	if err != nil {
+		log.Fatalf("Error parsing API URL: %v", err)
+	}
+
+	APIKey := "API-K6JR7YR2UTPH2BQZH1INRZUICEXHDF"
```
Comment (benPearce1) [inline src/shared-content/scripts/enable-disable-tenant-scripts.include.md:173]: should be updated to `API-YOUR-KEY`

File: src/shared-content/scripts/enable-disable-tenant-scripts.include.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+	"log"
+	"net/url"
+
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/client"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/spaces"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/tenants"
+)
+
+func main() {
+	apiURL, err := url.Parse("http://localhost:8066")
```
Comment (benPearce1) [inline src/shared-content/scripts/enable-disable-tenant-scripts.include.md:168]:
```suggestion
	apiURL, err := url.Parse("https://your-octopus-url")
```

File: src/shared-content/scripts/enable-disable-tenant-scripts.include.md (hunk 1 above)
Comment (benPearce1) [inline src/shared-content/scripts/enable-disable-tenant-scripts.include.md:173]:
```suggestion
	APIKey := "API-YOUR-KEY"
```

**Review by benPearce1 — APPROVED** [review]
couple of small tweaks

**Comment by steve-fenton-octopus** [comment]
One thing we ought to consider is that we're using words here from our [non-inclusive word list](https://www.octopus.design/932c0f1a9/p/457bc4-grammar-rules/t/page-457bc4-58679287-21).

If there's a way to switch to active/inactive tenants here, that would read far better. I'm not a fan of us ranking for "disabled tenants" or making folks uncomfortable with this term.

**Review by steve-fenton-octopus — APPROVED** [review]

**Pull request merged**